lightning-tag/implementation-setup-tasks.md

```diff
@@ -1,5 +1,28 @@
 # Pre-Implementation Setup
 
-!> Coming soon.
+Before deploying the Lotame Lightning Tag on your site, you must establish the following settings within the DMP:
 
-[comment]: # (Charlie is working on steps & screenshots.)
+1. **Domain Whitelist** - the list of domains on which Lightning Tag is allowed to collect data and utilize third-party cookies, first-party cookies, and localStorage.
+1. **Profile Extraction: Audience Owner Client** - The client id for which to provide audiences.
+1. **Profile Extraction: Audience Format** - Decide whether Lightning Tag will provide audiences as ids or targeting codes.
+
+## Domain Whitelisting
+
+For Lightning Tag to operate on your site, the domain must first be whitelisted. Domains can be added individually, or in bulk, directly into the DMP by navigating to ‘Client Admin: Manage Settings’ → ‘Domains.’
```

Comment:
For Lightning Tag to operate on your site, the domain must first be whitelisted. Domains can be whitelisted individually, or in bulk, directly within your DMP account by navigating to ‘Client Admin: Manage Settings’ → ‘Domains.’

Reply:
Will fix shortly.